install_files/ansible-base/roles/common/tasks/apply_grsec_lock.yml

```diff
@@ -32,11 +32,18 @@
       delay=45
       state=started
   when: not grsec_lock.stat.exists
+  # Local wait action, doesn't require sudo privileges.
```

Comment:
Since the top-level playbook (e.g. `securedrop-prod.yml`) sets `sudo: true`, all tasks run with sudo enabled unless it's explicitly overridden. Since the `wait_for` action means essentially "do nothing for a while", calling `sudo` is completely unnecessary.

If sudo if left as enabled here, the playbook will prompt for a sudo password when it hits this task. If the admin workstation has sudo configured, it can be typed in at this point, but I hardly think that's necessary. Worse, if you're running the staging machines via virtualbox and you use a yubikey for two-factor auth via the pam module, [sudoized ansible is fails altogether](https://github.com/ansible/ansible/issues/3969).